credentials/apps/edx_credentials_extensions/edly_credentials_app/api/v1/views/program_certificate_configuration.py

```diff
@@ -0,0 +1,44 @@
+"""
+Views for Edly Program Certificate Configuration API
+"""
+from rest_framework import status, viewsets
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.response import Response
+
+from credentials.apps.credentials.models import ProgramCertificate
+from credentials.apps.edx_credentials_extensions.edly_credentials_app.api.serializers import \
+    ProgramCertificateConfigurationSerializer
+
+
+class ProgramCertificateConfigurationViewSet(viewsets.ModelViewSet):
+    """
+    ViewSet for Program Certificate Configuration
+    """
+    lookup_field = 'program_uuid'
+    lookup_value_regex = '[0-9a-f-]+'
+    permission_classes = (IsAuthenticated,)
```

Comment:
IsAuthenticated permission is not enough. Could we add a permissions like:

1. Requesting user is `course creator` or `global course creator` (staff) 
2. Requesting user belongs to the site for which he/she is creating the program certificate

Reply:
Dibe